src/actionModifiers/ImportExportWithDatasets/importWithDatasetsModifier.js

```diff
@@ -164,6 +171,7 @@ function importWithDatasetsModifier(openmct) {
         datasets = undefined;
         referencedDatasets = undefined;
         component = undefined;
+        destroyForm = undefined;
```

Comment:
is this needed? I don't see it declared anywhere

Reply:
should be resetting `_destroy`. I've fixed.